solution.cpp

```diff
@@ -1,23 +1,121 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright &copy; 2015 Thomas Collins <thomas_collins@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
- */
+* Program that converts user inputted integers into Roman Numerals.
+*/
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+using std::endl;
+using std::cin;
+using std::cout;
+
+#include <string>
+using std::string;
+
+string intToRoman(int i);
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+int main()
+{
+	int i;
+	string s = "";
+	string repeat = "ayy lmao";
```

Comment:
random values that look like they have meaning, but really don't, are discouraged, lol (though things like this sometimes make their way into open source projects, as jokes -- search "linux kernel swear count" for an example, if you like).